### ISSUE TITLE
Saucelabs ios test timeout

### ISSUE THREAD

**Issue by dmydry** [webdriverio/webdriverio#1560]
## The problem

Running simple test:
```js
import { remote } from 'webdriverio';
const options = {
  desiredCapabilities: {
    browserName: 'safari',
    build: process.env.build,
    appiumVersion: '1.5.3',
    deviceName: 'iPhone 6 Simulator',
    deviceType: 'phone',
    deviceOrientation: 'portrait',
    platformVersion: '9.3',
    platformName: 'iOS',
  },
  user: 'xxx',
  key: 'xxx',
};
options.logLevel = 'verbose';
options.coloredLogs = true;
const client = remote(options);
client
  .init()
  .url('http://google.com');
```

## Environment

* WebdriverIO version: webdriverio@4.2.5
* Node.js version: v6.3.1
* [Standalone mode or wdio testrunner](http://webdriver.io/guide/getstarted/modes.html): stanalone

## Details

When I run this test there is a first test started on saucelabs. It takes some time (about 2 min) to run a vm there and start simulator. After simulator successfully started with safari, a second session appears there. Then, the first session failed (saucelabs error: Your test errored. The new session request was cancelled before a Sauce Labs virtual machine was found. For help, please check https://wiki.saucelabs.com/display/DOCS/Common+Error+Messages). And this can continue until I make my test process exit. Same test flow with android caps runs successful. 

I've tried to add some pause after init(). But no changes.

## Link to Selenium/WebdriverIO logs

Verbose logging shows me only these lines:
`[23:57:07]  COMMAND    	POST   	 "/wd/hub/session"`
`[23:57:07]  DATA       		{"desiredCapabilities":{"javascriptEnabled":true,"locationContextEnabled":true,"handlesAlerts":true,"rotatable":true,"browserName":"safari","appiumVersion":"1.5.3","deviceName":"iPhone Simulator","deviceType":"phone","deviceOrientation":"portrait","platformVersion":"9.3","_":"4 more keys: [\"platformName\",\"name\",\"idleTimeout\",\"requestOrigins\"]"}}`

But it should catch a session info to continue, if I get it right, like this:
`[23:05:46]  INFO       	SET SESSION ID 251b83e895fd4e88ad6123ea91694f59` (saved from android run)

**Comment by vvo** [comment]
Same issue for me, using platform configurator and putting those in webdriverio results in failure in saucelabs (timeout/no vm started).

This seems to be a saucelabs issue. Recurring one with mobile.

**Comment by alodium** [comment]
Same here, ios emulator test.

**Comment by dmydry** [comment]
I've sent an email to saucelabs support with a link to this report. Let's see if they react

**Comment by zinserjan** [comment]
I had the same error a few months ago. I was able to fix this with an increased timeout for `connectionRetryTimeout`.

Normally it takes around ~90s to boot up an iOS simulator via saucelabs completely. Worst case for me was around 150s. WebdriverIO uses a default connection timeout of 90s and this leads to your described problem with aborted and recreated sessions.

I just doubled the value to 180s and had no further problems.
